### PR TITLE
Apply next-routes

### PR DESCRIPTION
**Pull request by ooookai**

### CODE REVIEW

**Comment by ooookai** [comment]
issue #15

File: pages/reply.js [hunk 1]
```diff
@@ -78,10 +75,8 @@ class ReplyPage extends React.Component {
       : '查看文章頁面';
 
     return (
-      <Link
-        href={`/article?id=${originalArticle.get('id')}`}
-        as={`/article/${originalArticle.get('id')}`}
-      >
+      <Link route='article' params={{id: article.get('id')}}>
```
Comment (MrOrz) [inline pages/reply.js]: Running `yarn lint` should tell you that `article` is not defined. Please help fix this.

File: server.js [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
 
 const app = next({ dev: process.env.NODE_ENV !== 'production' });
 const handle = app.getRequestHandler();
 
+const routes = require('./routes')
+const handler = routes.getRequestHandler(app)
+
 // viewPath: mapped client-side route
 //
 const render = viewPath => ctx => {
```
Comment (MrOrz) [inline server.js]: Please run `yarn lint` to find out unused vars and remove them.

**Comment by ooookai** [comment]
Yeah, Travis check failed because there is no Travis setup on this branch. (?

**Pull request merged**